Loading cifar10 dataset.
Files already downloaded and verified
Files already downloaded and verified
Files already downloaded and verified
Creating lottery-vgg16 model.
Pre-Train for 0 epochs.
Traceback (most recent call last):
  File "main.py", line 135, in <module>
    singleshot.run(args)
  File "/home/vikashm/COS598D/COS598D-Pruning/Experiments/singleshot.py", line 39, in run
    test_loader, device, args.pre_epochs, args.verbose)
TypeError: train_eval_loop() missing 1 required positional argument: 'graph_name'
Loading cifar10 dataset.
Files already downloaded and verified
Files already downloaded and verified
Files already downloaded and verified
Creating lottery-vgg16 model.
Pre-Train for 0 epochs.
Traceback (most recent call last):
  File "main.py", line 135, in <module>
    singleshot.run(args)
  File "/home/vikashm/COS598D/COS598D-Pruning/Experiments/singleshot.py", line 39, in run
    test_loader, device, args.pre_epochs, args.verbose)
TypeError: train_eval_loop() missing 1 required positional argument: 'graph_name'
Loading cifar10 dataset.
Files already downloaded and verified
Files already downloaded and verified
Files already downloaded and verified
Creating lottery-vgg16 model.
Pre-Train for 0 epochs.
Traceback (most recent call last):
  File "main.py", line 135, in <module>
    singleshot.run(args)
  File "/home/vikashm/COS598D/COS598D-Pruning/Experiments/singleshot.py", line 39, in run
    test_loader, device, args.pre_epochs, args.verbose)
TypeError: train_eval_loop() missing 1 required positional argument: 'graph_name'
Loading cifar10 dataset.
Files already downloaded and verified
Files already downloaded and verified
Files already downloaded and verified
Creating lottery-vgg16 model.
Pre-Train for 0 epochs.
Traceback (most recent call last):
  File "main.py", line 135, in <module>
    singleshot.run(args)
  File "/home/vikashm/COS598D/COS598D-Pruning/Experiments/singleshot.py", line 39, in run
    test_loader, device, args.pre_epochs, args.verbose)
TypeError: train_eval_loop() missing 1 required positional argument: 'graph_name'
Loading cifar10 dataset.
Files already downloaded and verified
Files already downloaded and verified
Files already downloaded and verified
Creating lottery-vgg16 model.
Pre-Train for 0 epochs.
Traceback (most recent call last):
  File "main.py", line 135, in <module>
    singleshot.run(args)
  File "/home/vikashm/COS598D/COS598D-Pruning/Experiments/singleshot.py", line 39, in run
    test_loader, device, args.pre_epochs, args.verbose)
TypeError: train_eval_loop() missing 1 required positional argument: 'graph_name'
Loading mnist dataset.
Creating default-fc model.
Pre-Train for 0 epochs.
Traceback (most recent call last):
  File "main.py", line 135, in <module>
    singleshot.run(args)
  File "/home/vikashm/COS598D/COS598D-Pruning/Experiments/singleshot.py", line 39, in run
    test_loader, device, args.pre_epochs, args.verbose)
TypeError: train_eval_loop() missing 1 required positional argument: 'graph_name'
Loading mnist dataset.
Creating default-fc model.
Pre-Train for 0 epochs.
Traceback (most recent call last):
  File "main.py", line 135, in <module>
    singleshot.run(args)
  File "/home/vikashm/COS598D/COS598D-Pruning/Experiments/singleshot.py", line 39, in run
    test_loader, device, args.pre_epochs, args.verbose)
TypeError: train_eval_loop() missing 1 required positional argument: 'graph_name'
Loading mnist dataset.
Creating default-fc model.
Pre-Train for 0 epochs.
Traceback (most recent call last):
  File "main.py", line 135, in <module>
    singleshot.run(args)
  File "/home/vikashm/COS598D/COS598D-Pruning/Experiments/singleshot.py", line 39, in run
    test_loader, device, args.pre_epochs, args.verbose)
TypeError: train_eval_loop() missing 1 required positional argument: 'graph_name'
Loading mnist dataset.
Creating default-fc model.
Pre-Train for 0 epochs.
Traceback (most recent call last):
  File "main.py", line 135, in <module>
    singleshot.run(args)
  File "/home/vikashm/COS598D/COS598D-Pruning/Experiments/singleshot.py", line 39, in run
    test_loader, device, args.pre_epochs, args.verbose)
TypeError: train_eval_loop() missing 1 required positional argument: 'graph_name'
Loading mnist dataset.
Creating default-fc model.
Pre-Train for 0 epochs.
Traceback (most recent call last):
  File "main.py", line 135, in <module>
    singleshot.run(args)
  File "/home/vikashm/COS598D/COS598D-Pruning/Experiments/singleshot.py", line 39, in run
    test_loader, device, args.pre_epochs, args.verbose)
TypeError: train_eval_loop() missing 1 required positional argument: 'graph_name'
Loading cifar10 dataset.
Files already downloaded and verified
Files already downloaded and verified
Files already downloaded and verified
Creating lottery-vgg16 model.
Pre-Train for 0 epochs.
Traceback (most recent call last):
  File "main.py", line 135, in <module>
    singleshot.run(args)
  File "/home/vikashm/COS598D/COS598D-Pruning/Experiments/singleshot.py", line 39, in run
    test_loader, device, args.pre_epochs, args.verbose)
TypeError: train_eval_loop() missing 1 required positional argument: 'graph_name'
Loading cifar10 dataset.
Files already downloaded and verified
Files already downloaded and verified
Files already downloaded and verified
Creating lottery-vgg16 model.
Pre-Train for 0 epochs.
Traceback (most recent call last):
  File "main.py", line 135, in <module>
    singleshot.run(args)
  File "/home/vikashm/COS598D/COS598D-Pruning/Experiments/singleshot.py", line 39, in run
    test_loader, device, args.pre_epochs, args.verbose)
TypeError: train_eval_loop() missing 1 required positional argument: 'graph_name'
Loading cifar10 dataset.
Files already downloaded and verified
Files already downloaded and verified
Files already downloaded and verified
Creating lottery-vgg16 model.
Pre-Train for 0 epochs.
Traceback (most recent call last):
  File "main.py", line 135, in <module>
    singleshot.run(args)
  File "/home/vikashm/COS598D/COS598D-Pruning/Experiments/singleshot.py", line 39, in run
    test_loader, device, args.pre_epochs, args.verbose)
TypeError: train_eval_loop() missing 1 required positional argument: 'graph_name'
Loading cifar10 dataset.
Files already downloaded and verified
Files already downloaded and verified
Files already downloaded and verified
Creating lottery-vgg16 model.
Pre-Train for 0 epochs.
Traceback (most recent call last):
  File "main.py", line 135, in <module>
    singleshot.run(args)
  File "/home/vikashm/COS598D/COS598D-Pruning/Experiments/singleshot.py", line 39, in run
    test_loader, device, args.pre_epochs, args.verbose)
TypeError: train_eval_loop() missing 1 required positional argument: 'graph_name'
Loading cifar10 dataset.
Files already downloaded and verified
Files already downloaded and verified
Files already downloaded and verified
Creating lottery-vgg16 model.
Pre-Train for 0 epochs.
Traceback (most recent call last):
  File "main.py", line 135, in <module>
    singleshot.run(args)
  File "/home/vikashm/COS598D/COS598D-Pruning/Experiments/singleshot.py", line 39, in run
    test_loader, device, args.pre_epochs, args.verbose)
TypeError: train_eval_loop() missing 1 required positional argument: 'graph_name'
Loading cifar10 dataset.
Files already downloaded and verified
Files already downloaded and verified
Files already downloaded and verified
Creating lottery-vgg16 model.
Pre-Train for 0 epochs.
Traceback (most recent call last):
  File "main.py", line 135, in <module>
    singleshot.run(args)
  File "/home/vikashm/COS598D/COS598D-Pruning/Experiments/singleshot.py", line 39, in run
    test_loader, device, args.pre_epochs, args.verbose)
TypeError: train_eval_loop() missing 1 required positional argument: 'graph_name'
Loading cifar10 dataset.
Files already downloaded and verified
Files already downloaded and verified
Files already downloaded and verified
Creating lottery-vgg16 model.
Pre-Train for 0 epochs.
Traceback (most recent call last):
  File "main.py", line 135, in <module>
    singleshot.run(args)
  File "/home/vikashm/COS598D/COS598D-Pruning/Experiments/singleshot.py", line 39, in run
    test_loader, device, args.pre_epochs, args.verbose)
TypeError: train_eval_loop() missing 1 required positional argument: 'graph_name'
Loading cifar10 dataset.
Files already downloaded and verified
Files already downloaded and verified
Files already downloaded and verified
Creating lottery-vgg16 model.
Pre-Train for 0 epochs.
Traceback (most recent call last):
  File "main.py", line 135, in <module>
    singleshot.run(args)
  File "/home/vikashm/COS598D/COS598D-Pruning/Experiments/singleshot.py", line 39, in run
    test_loader, device, args.pre_epochs, args.verbose)
TypeError: train_eval_loop() missing 1 required positional argument: 'graph_name'
Loading cifar10 dataset.
Files already downloaded and verified
Files already downloaded and verified
Files already downloaded and verified
Creating lottery-vgg16 model.
Pre-Train for 0 epochs.
Traceback (most recent call last):
  File "main.py", line 135, in <module>
    singleshot.run(args)
  File "/home/vikashm/COS598D/COS598D-Pruning/Experiments/singleshot.py", line 39, in run
    test_loader, device, args.pre_epochs, args.verbose)
TypeError: train_eval_loop() missing 1 required positional argument: 'graph_name'
Loading cifar10 dataset.
Files already downloaded and verified
Files already downloaded and verified
Files already downloaded and verified
Creating lottery-vgg16 model.
Pre-Train for 0 epochs.
Traceback (most recent call last):
  File "main.py", line 135, in <module>
    singleshot.run(args)
  File "/home/vikashm/COS598D/COS598D-Pruning/Experiments/singleshot.py", line 39, in run
    test_loader, device, args.pre_epochs, args.verbose)
TypeError: train_eval_loop() missing 1 required positional argument: 'graph_name'
Loading cifar10 dataset.
Files already downloaded and verified
Files already downloaded and verified
Files already downloaded and verified
Creating lottery-vgg16 model.
Pre-Train for 0 epochs.
Traceback (most recent call last):
  File "main.py", line 135, in <module>
    singleshot.run(args)
  File "/home/vikashm/COS598D/COS598D-Pruning/Experiments/singleshot.py", line 39, in run
    test_loader, device, args.pre_epochs, args.verbose)
TypeError: train_eval_loop() missing 1 required positional argument: 'graph_name'
Loading cifar10 dataset.
Files already downloaded and verified
Files already downloaded and verified
Files already downloaded and verified
Creating lottery-vgg16 model.
Pre-Train for 0 epochs.
Traceback (most recent call last):
  File "main.py", line 135, in <module>
    singleshot.run(args)
  File "/home/vikashm/COS598D/COS598D-Pruning/Experiments/singleshot.py", line 39, in run
    test_loader, device, args.pre_epochs, args.verbose)
TypeError: train_eval_loop() missing 1 required positional argument: 'graph_name'
Loading cifar10 dataset.
Files already downloaded and verified
Files already downloaded and verified
Files already downloaded and verified
Creating lottery-vgg16 model.
Pre-Train for 0 epochs.
Traceback (most recent call last):
  File "main.py", line 135, in <module>
    singleshot.run(args)
  File "/home/vikashm/COS598D/COS598D-Pruning/Experiments/singleshot.py", line 39, in run
    test_loader, device, args.pre_epochs, args.verbose)
TypeError: train_eval_loop() missing 1 required positional argument: 'graph_name'
Loading cifar10 dataset.
Files already downloaded and verified
Files already downloaded and verified
Files already downloaded and verified
Creating lottery-vgg16 model.
Pre-Train for 0 epochs.
Traceback (most recent call last):
  File "main.py", line 135, in <module>
    singleshot.run(args)
  File "/home/vikashm/COS598D/COS598D-Pruning/Experiments/singleshot.py", line 39, in run
    test_loader, device, args.pre_epochs, args.verbose)
TypeError: train_eval_loop() missing 1 required positional argument: 'graph_name'
Loading cifar10 dataset.
Files already downloaded and verified
Files already downloaded and verified
Files already downloaded and verified
Creating lottery-vgg16 model.
Pre-Train for 0 epochs.
Traceback (most recent call last):
  File "main.py", line 135, in <module>
    singleshot.run(args)
  File "/home/vikashm/COS598D/COS598D-Pruning/Experiments/singleshot.py", line 39, in run
    test_loader, device, args.pre_epochs, args.verbose)
TypeError: train_eval_loop() missing 1 required positional argument: 'graph_name'
Loading cifar10 dataset.
Files already downloaded and verified
Files already downloaded and verified
Files already downloaded and verified
Creating lottery-vgg16 model.
Pre-Train for 0 epochs.
Traceback (most recent call last):
  File "main.py", line 135, in <module>
    singleshot.run(args)
  File "/home/vikashm/COS598D/COS598D-Pruning/Experiments/singleshot.py", line 39, in run
    test_loader, device, args.pre_epochs, args.verbose)
TypeError: train_eval_loop() missing 1 required positional argument: 'graph_name'
Loading cifar10 dataset.
Files already downloaded and verified
Files already downloaded and verified
Files already downloaded and verified
Creating lottery-vgg16 model.
Pre-Train for 0 epochs.
Traceback (most recent call last):
  File "main.py", line 135, in <module>
    singleshot.run(args)
  File "/home/vikashm/COS598D/COS598D-Pruning/Experiments/singleshot.py", line 39, in run
    test_loader, device, args.pre_epochs, args.verbose)
TypeError: train_eval_loop() missing 1 required positional argument: 'graph_name'
Loading cifar10 dataset.
Files already downloaded and verified
Files already downloaded and verified
Files already downloaded and verified
Creating lottery-vgg16 model.
Pre-Train for 0 epochs.
Traceback (most recent call last):
  File "main.py", line 135, in <module>
    singleshot.run(args)
  File "/home/vikashm/COS598D/COS598D-Pruning/Experiments/singleshot.py", line 39, in run
    test_loader, device, args.pre_epochs, args.verbose)
TypeError: train_eval_loop() missing 1 required positional argument: 'graph_name'
Loading cifar10 dataset.
Files already downloaded and verified
Files already downloaded and verified
Files already downloaded and verified
Creating lottery-vgg16 model.
Pre-Train for 0 epochs.
Traceback (most recent call last):
  File "main.py", line 135, in <module>
    singleshot.run(args)
  File "/home/vikashm/COS598D/COS598D-Pruning/Experiments/singleshot.py", line 39, in run
    test_loader, device, args.pre_epochs, args.verbose)
TypeError: train_eval_loop() missing 1 required positional argument: 'graph_name'
Loading cifar10 dataset.
Files already downloaded and verified
Files already downloaded and verified
Files already downloaded and verified
Creating lottery-vgg16 model.
Pre-Train for 0 epochs.
Traceback (most recent call last):
  File "main.py", line 135, in <module>
    singleshot.run(args)
  File "/home/vikashm/COS598D/COS598D-Pruning/Experiments/singleshot.py", line 39, in run
    test_loader, device, args.pre_epochs, args.verbose)
TypeError: train_eval_loop() missing 1 required positional argument: 'graph_name'
Loading cifar10 dataset.
Files already downloaded and verified
Files already downloaded and verified
Files already downloaded and verified
Creating lottery-vgg16 model.
Pre-Train for 0 epochs.
Traceback (most recent call last):
  File "main.py", line 135, in <module>
    singleshot.run(args)
  File "/home/vikashm/COS598D/COS598D-Pruning/Experiments/singleshot.py", line 39, in run
    test_loader, device, args.pre_epochs, args.verbose)
TypeError: train_eval_loop() missing 1 required positional argument: 'graph_name'
Loading cifar10 dataset.
Files already downloaded and verified
Files already downloaded and verified
Files already downloaded and verified
Creating lottery-vgg16 model.
Pre-Train for 0 epochs.
Traceback (most recent call last):
  File "main.py", line 135, in <module>
    singleshot.run(args)
  File "/home/vikashm/COS598D/COS598D-Pruning/Experiments/singleshot.py", line 39, in run
    test_loader, device, args.pre_epochs, args.verbose)
TypeError: train_eval_loop() missing 1 required positional argument: 'graph_name'
Loading cifar10 dataset.
Files already downloaded and verified
Files already downloaded and verified
Files already downloaded and verified
Creating lottery-vgg16 model.
Pre-Train for 0 epochs.
Traceback (most recent call last):
  File "main.py", line 135, in <module>
    singleshot.run(args)
  File "/home/vikashm/COS598D/COS598D-Pruning/Experiments/singleshot.py", line 39, in run
    test_loader, device, args.pre_epochs, args.verbose)
TypeError: train_eval_loop() missing 1 required positional argument: 'graph_name'
Experiment 'singleshot' with expid 'hp-grasp-1' exists.  Overwrite (yes/no)? Traceback (most recent call last):
  File "main.py", line 120, in <module>
    os.makedirs(result_dir)
  File "/usr/lib64/python3.6/os.py", line 220, in makedirs
    mkdir(name, mode)
FileExistsError: [Errno 17] File exists: '/scratch/network/vikashm/Results/data_sparsity_order2/singleshot/hp-grasp-1'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "main.py", line 124, in <module>
    val = input("Experiment '{}' with expid '{}' exists.  Overwrite (yes/no)? ".format(args.experiment, args.expid))
EOFError: EOF when reading a line
Experiment 'singleshot' with expid 'hp-synflow-1' exists.  Overwrite (yes/no)? Traceback (most recent call last):
  File "main.py", line 120, in <module>
    os.makedirs(result_dir)
  File "/usr/lib64/python3.6/os.py", line 220, in makedirs
    mkdir(name, mode)
FileExistsError: [Errno 17] File exists: '/scratch/network/vikashm/Results/data_sparsity_order2/singleshot/hp-synflow-1'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "main.py", line 124, in <module>
    val = input("Experiment '{}' with expid '{}' exists.  Overwrite (yes/no)? ".format(args.experiment, args.expid))
EOFError: EOF when reading a line
Experiment 'singleshot' with expid 'hp-rand-2' exists.  Overwrite (yes/no)? Traceback (most recent call last):
  File "main.py", line 120, in <module>
    os.makedirs(result_dir)
  File "/usr/lib64/python3.6/os.py", line 220, in makedirs
    mkdir(name, mode)
FileExistsError: [Errno 17] File exists: '/scratch/network/vikashm/Results/data_sparsity_order2/singleshot/hp-rand-2'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "main.py", line 124, in <module>
    val = input("Experiment '{}' with expid '{}' exists.  Overwrite (yes/no)? ".format(args.experiment, args.expid))
EOFError: EOF when reading a line
Experiment 'singleshot' with expid 'hp-mag-2' exists.  Overwrite (yes/no)? Traceback (most recent call last):
  File "main.py", line 120, in <module>
    os.makedirs(result_dir)
  File "/usr/lib64/python3.6/os.py", line 220, in makedirs
    mkdir(name, mode)
FileExistsError: [Errno 17] File exists: '/scratch/network/vikashm/Results/data_sparsity_order2/singleshot/hp-mag-2'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "main.py", line 124, in <module>
    val = input("Experiment '{}' with expid '{}' exists.  Overwrite (yes/no)? ".format(args.experiment, args.expid))
EOFError: EOF when reading a line
Experiment 'singleshot' with expid 'hp-snip-2' exists.  Overwrite (yes/no)? Traceback (most recent call last):
  File "main.py", line 120, in <module>
    os.makedirs(result_dir)
  File "/usr/lib64/python3.6/os.py", line 220, in makedirs
    mkdir(name, mode)
FileExistsError: [Errno 17] File exists: '/scratch/network/vikashm/Results/data_sparsity_order2/singleshot/hp-snip-2'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "main.py", line 124, in <module>
    val = input("Experiment '{}' with expid '{}' exists.  Overwrite (yes/no)? ".format(args.experiment, args.expid))
EOFError: EOF when reading a line
Experiment 'singleshot' with expid 'hp-grasp-2' exists.  Overwrite (yes/no)? Traceback (most recent call last):
  File "main.py", line 120, in <module>
    os.makedirs(result_dir)
  File "/usr/lib64/python3.6/os.py", line 220, in makedirs
    mkdir(name, mode)
FileExistsError: [Errno 17] File exists: '/scratch/network/vikashm/Results/data_sparsity_order2/singleshot/hp-grasp-2'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "main.py", line 124, in <module>
    val = input("Experiment '{}' with expid '{}' exists.  Overwrite (yes/no)? ".format(args.experiment, args.expid))
EOFError: EOF when reading a line
Experiment 'singleshot' with expid 'hp-synflow-2' exists.  Overwrite (yes/no)? Traceback (most recent call last):
  File "main.py", line 120, in <module>
    os.makedirs(result_dir)
  File "/usr/lib64/python3.6/os.py", line 220, in makedirs
    mkdir(name, mode)
FileExistsError: [Errno 17] File exists: '/scratch/network/vikashm/Results/data_sparsity_order2/singleshot/hp-synflow-2'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "main.py", line 124, in <module>
    val = input("Experiment '{}' with expid '{}' exists.  Overwrite (yes/no)? ".format(args.experiment, args.expid))
EOFError: EOF when reading a line
